```
FAILED tests/test_nonequilibrium_cycling.py::TestNonEquilibriumCycling::test_partial_charge_assignation[am1bcc-openeye] - ImportError: OpenEye is not available and cannot be selected as a backend
FAILED tests/test_nonequilibrium_cycling.py::TestNonEquilibriumCycling::test_partial_charge_assignation[am1bccelf10-openeye] - ImportError: OpenEye is not available and cannot be selected as a backend
FAILED tests/test_nonequilibrium_cycling.py::TestNonEquilibriumCycling::test_partial_charge_assignation[espaloma-ambertools] - ImportError: The Espaloma ToolkiWrapper is not available, please install espaloma_charge
FAILED tests/test_nonequilibrium_cycling.py::TestNonEquilibriumCycling::test_partial_charge_assignation[espaloma-rdkit] - ImportError: The Espaloma ToolkiWrapper is not available, please install espaloma_charge
``` 
If we want to run CI as part of our conda release, we will need to set these tests to skip if we are missing packages. We could test espaloma but the opene-eye ones will not work.

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,6 +36,7 @@ test:
   imports:
     - feflow
   requires:
+    - python {{ python_min }}
 
 about:
   home: https://github.com/OpenFreeEnergy/feflow

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,12 +36,6 @@ test:
   imports:
     - feflow
   requires:
-    - openmmforcefields
-    - pytest
-    - pytest-xdist
-    - python {{ python_min }}
-  commands:
-    - pytest -v -n logical --pyargs feflow
 
 about:
   home: https://github.com/OpenFreeEnergy/feflow

